.github/workflows/cd.yml

```diff
@@ -0,0 +1,51 @@
+name: Business Filing UI CD
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+    inputs:
+      environement:
+        description: "Environment (dev/test)"
+        required: true
+        default: "dev"
+
+jobs:
+  business-filing-ui-dev:
+    runs-on: ubuntu-latest
+    if: github.repository == 'bcgov/business-filings-ui'
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set env by input
+        run: |
+          echo "::set-env name=TAG_NAME::${{ github.event.inputs.environement }}"
+
+      - name: Set default env by master branch
+        if: env.TAG_NAME == '' && github.ref == 'refs/heads/master'
+        run: |
+          echo "::set-env name=TAG_NAME::dev"
+
+      - uses: pwei1018/bcrs-cd-action@latest
```

Comment:
Path?

Reply:
The composite action  is a new thing.  So, I just put it into my own repo. I have other change plan of this action waiting for the new features release from github.

Reply:
Very cool.
I'd feel more comfortable if this used a bcgov repo even now, but I'll leave it up to you.